services/cbioproxy/curl_studyIds_from_portal.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# curling studyIds from api
+CURLSTRING=$(curl http://cbioportal:8080/api/studies)
```

Comment:
Is this endpoint always public? Even when cBioPortal is connected to keycloak using SAML?]
Sourcecode-wise it looks fine, just asking because I can't test this right now.

Reply:
ah, you are right, this could be a problem indeed ;) 
I will instead try to include the curl command in the lua code itself, then the sessionid of the user can be sent with it...